src/components/forms/login-form.js

```diff
@@ -1,7 +1,7 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import gql from 'graphql-tag';
-import NextLink from 'next/link';
+// import { Link as RouterLink } from 'react-router-dom';
```

Comment:
this should be deleted I suppose?

Reply:
we may want to remember what we did before when time has been come.